### PR TITLE
Burhan c18 js2 w2

### PR DESCRIPTION
**Pull request by burhankhuzam**
JS2 Week 2 Homework

### CODE REVIEW

File: Week2/homework/maartjes_work.js [hunk 1]
```diff
@@ -1,47 +1,30 @@
...
+  ];
+  const tasks = monday.concat(tuesday);
 
-// Add your code here
+  let twoDaysIncome = tasks
+    .map(task => task.duration / 60)
+    .filter(task => task >= 2)
+    .map(total => total * 60)
+    .reduce((a, b) => a + b, 0);
+  console.log("Maartje earned " +twoDaysIncome+ ".00 Euro on Monday and Tuesday!");
```
Comment (rohni) [inline Week2/homework/maartjes_work.js]: Good job, it is all there, I would only mention that there are ways to have javascript do the string formatting for you.  You might want to look up the .toLocaleString string method.

File: Week2/homework/maartjes_work.js [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
+      duration: 10 },
+    { name: 'Talk to a lot of people',
+      duration: 200 },
+    { name: 'Look at application assignments new students',
+      duration: 40 }
+  ];
+  const tasks = monday.concat(tuesday);
 
-// Add your code here
+  let twoDaysIncome = tasks
```
Comment (rohni) [inline Week2/homework/maartjes_work.js]: I like the way you have neatly chained all the operations on their own line. Makes it very readable.

File: Week2/homework/map_filter.js [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+//   if (numbers[i] % 2 !== 0) {
+//     newNumbers.push(numbers[i] * 2);
+//   }
+// }
 
-// Add your code here
+let newNumbers = numbers
+    .filter(number => number % 2 !== 0) 
+    .map(number => number * 2);
+console.log('The doubled numbers are', newNumbers);
```
Comment (rohni) [inline Week2/homework/map_filter.js]: Good job.  As an extra challenge.  How would you implement this with Array.reduce? 😄

Comment (burhankhuzam) [reply Week2/homework/map_filter.js]: Hi Rohni,
Thank for spending time to review my homework!

Comment (burhankhuzam) [reply Week2/homework/map_filter.js]: I honestly have a memory issue...
I could not remember what was the way of thinking I followed last week...
Maybe I need to refresh my my thoughts with some tutorial videos and will replay to you asap!
Thanks again!!!

**Pull request closed without merging**